src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -501,6 +511,14 @@ public interface IStartConfiguration
             /// No update checks or analytics collection should be done.
             /// </summary>
             bool IsHeadless { get; set; }
+            /// <summary>
+            /// Default Python script engine
+            /// </summary>
+            string DefaultPythonEngine { get; set; }
```

Comment:
modifying this interface is an API break - we can add a new interface instead, and merge them together in 3.x

Reply:
An alternative can be not have this in the interface but keep it in `DefaultStartConfiguration`, and cast and see in `DynamoModel` constructor. I see some other properties on this boat already.